core/block_svg.js

```diff
@@ -69,6 +69,13 @@ Blockly.BlockSvg = function(workspace, prototypeName, opt_id) {
 
   /** @type {boolean} */
   this.rendered = false;
+  /**
+   * Is this block currently rendering? Used to stop recursive render calls.
```

Comment:
Is it accurate to say we are stopping recursive render calls or is it that we are stopping recursive cycle? 
This may be right, I'm just not sure.

Reply:
How bouts "Used to stop recursive render calls from actually triggering a re-render"?

Reply:
lgtm